.ci/oci-launch-e2e.sh

```diff
@@ -12,18 +12,5 @@ export WORKSPACE=$(dirname $(dirname $(readlink -f "$0")));
 # Available openshift ci environments https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables
 export ARTIFACTS_DIR=${ARTIFACT_DIR:-"/tmp/appstudio"}
 
-function executeE2ETests() {
```

Comment:
@flacatus  is there a place in CI where we are still using this `.ci/oci-launch-e2e.sh` script?